nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/AbstractJMSProcessor.java

```diff
@@ -196,7 +207,10 @@ private void buildTargetResource(ProcessContext context) {
 
             this.cachingConnectionFactory = new CachingConnectionFactory(cfCredentialsAdapter);
             this.cachingConnectionFactory.setSessionCacheSize(Integer.parseInt(context.getProperty(SESSION_CACHE_SIZE).getValue()));
-
+            String clientId = context.getProperty(CLIENT_ID).getValue();
```

Comment:
We'd like to evaluate expression here. Currently unit test fails because it doesn't evaluate expression even if it supports EL.